js/drawmap_ns.js

```diff
@@ -102,16 +103,34 @@ function drawPoints(R, somePoints, baseX, baseY) {
         seen[b.name] = true;
 
         // Map point:
-        var p = R.circle(b.x, b.y, 6).attr({stroke:b.stroke, fill:b.fill, "stroke-width": 2, "href": b.url, opacity: opacity});
-        // var p = R.circle(b.x, b.y, 6).attr({stroke:b.stroke, fill:b.fill, "stroke-width": 2, opacity: opacity});
+        var p;
+        if (isDebug()) {
+            p = R.circle(b.x, b.y, 6).attr({stroke:b.stroke, fill:b.fill, "stroke-width": 2, opacity: opacity});
```

Comment:
Please add a comment here saying that in debug mode, we disable the clickable link from dot -> brewery url and enable the dots to be dragged around (and new x,y printed to the console)